src/components/Input.tsx

```diff
@@ -0,0 +1,32 @@
+import { EyeIcon } from "@heroicons/react/24/solid";
+
+type InputProps = {
+	type: string;
+};
+
+export default function Input(props: InputProps) {
+	let placeholder = "";
```

Comment:
use can make that more concise by just using a ternary here to compute the value

Reply:
true, done